pkg/controller/subscription/subscription_controller.go

```diff
@@ -250,6 +251,16 @@ func (r *ReconcileSubscription) Reconcile(ctx context.Context, request reconcile
 				klog.Errorf("doReconcile got error %v", reconcileErr)
 			}
 
+			// Update AppstatusReference
+			if !r.standalone {
```

Comment:
Does it mean the AppStatus reference is only visible for non standalone appsub?  It looks not necessary.

For standalone appsub, It looks the AppStatusreference is still valid. I just checked some standalone appsub e.g
```
% oc get appsub -n open-cluster-management          cluster-lifecycle-sub          
NAME                    STATUS       AGE     LOCAL PLACEMENT   TIME WINDOW
cluster-lifecycle-sub   Subscribed   5h11m   true      
```
And its appsubstatus is 
```
% oc get appsubstatus -n open-cluster-management          cluster-lifecycle-sub 
NAME                    AGE
cluster-lifecycle-sub   5h12m
```

Reply:
XJ, you're right with the standalone case. Let me update that.